ucgmsim/gmhazard
# Fix the NZS1170.5 test data and test itself

# Fix the NZS1170.5 test data and test itself

- Update the logic to check both radius 0 and 2.
- Update the test data as the soil class is now `soft_or_deep_soil` when Vs30 < 300 instead of < 350.
